packages/palette/src/elements/ShowMore/ShowMore.tsx

```diff
@@ -14,6 +18,8 @@ export const ShowMore: React.FC<ShowMoreProps> = ({
   initial = INITIAL_ITEMS_TO_SHOW,
   children,
   expanded = false,
+  clickableProps = { mt: "1", textDecoration: "underline", textAlign: "left" },
+  textProps = { variant: "xs" },
```

Comment:
This is clunky/ugly. We have a utility that lets you split off props so that you can just use `...rest` https://github.com/artsy/palette/blob/main/packages/palette/src/utils/splitProps.ts

Reply:
(Though; I can see that being a lil odd to pull off in practice.)

Reply:
It's less about disallowing and more about not letting the props API become a mess. As an aside this seems like a not well designed component/not really appropriate for Palette. But that's another story.

Reply:
Like: it'd make more sense to simply remove _all_ text styling and let it inherit from its parent. Since it's not 'designed'

Reply:
> As an aside this seems like a not well designed component/not really appropriate for Palette. But that's another story.

This got me wondering about where it's used - turns out it's not actually being consumed by any components outside of Palette (within Palette, it's used for `FilterSelect`, and I see you & @damassi [butted heads](https://github.com/artsy/palette/pull/1099) over that choice when it was introduced). Force has its [own version](https://github.com/artsy/force/blob/main/src/Components/ArtworkFilter/ArtworkFilters/ShowMore.tsx) of the component, which I assume is where this one came from in the first place.

I take your point about whether it makes sense as part of our design system. We should either make it a first-class component (basically what this PR does, though maybe it would also need more thought around use-cases), or make it inaccessible outside of Palette and let clients create their own.

IMO it's still a useful "util" component (want to hide/show a few elements easily? done), but this isn't a hill I'm prepared to die on. If you still think it's an anti-pattern, I'll update this PR to remove it from exported components and do something similar in Force.

Reply:
I'd take @dzucconi's design sys guidance over mine any day 😄 

It feels like a read more should be in palette, just because we use it all over, but yeah I agree with him that ideally it wouldn't be styled; and / or it could be more clever in that way. So far it has worked well enough tho!